tm2/pkg/bft/config/config.go

```diff
@@ -202,12 +192,6 @@ type BaseConfig struct {
 	// Database directory
 	DBPath string `toml:"db_dir"`
 
-	// Output level for logging
-	LogLevel string `toml:"log_level"`
-
-	// Output format: 'plain' (colored text) or 'json'
-	LogFormat string `toml:"log_format"`
```

Comment:
I think we can and should preserve this behavior.

Reply:
I'm on the fence about this, and would like to discuss. The reason I've moved it out of the config is because right now, there is absolutely no way to set (even the base) config params as someone who's initializing the node (we just don't have the command suite for it), which leaves users and node operators to manually change this value in the TOML.

I'm a believer that these runtime parameters (log level, listen port, log output...) shouldn't be set in a config that can't be changed easily.

The options I see here are:
1. Leave the log level / format to be specified in the config TOML, making it hard to change (you need to do it manually)
2. Leave the log level / format to be specified as flags, but eventually migrate it to the config file (when we add the command suite for it)
3. Leave the log level / format to be specified as flags, dropping it from the config file altogether

I would like you hear what you think about this, it's a design choice we need to make, and it affects how we treat other functionality as well.

cc @gfanton @thehowl @ajnavarro